File: dropwizard-archetypes/java-simple/src/main/resources/archetype-resources/src/main/java/__name__Service.java
package ${package};

import com.codahale.dropwizard.Service;
import com.codahale.dropwizard.setup.Bootstrap;
import com.codahale.dropwizard.setup.Environment;

public class ${name}Service extends Service<${name}Configuration> {

    public static void main(final String[] args) throws Exception {
        new ${name}Service().run(args);
    }

    @Override
    public String getName() {
        return "${name}";
    }

    @Override
    public void initialize(final Bootstrap<${name}Configuration> bootstrap) {
        // TODO: service initialization
    }

    @Override
    public void run(final ${name}Configuration configuration,
                    final Environment environment) {
        // TODO: implement service
    }

}

File: dropwizard-archetypes/scala-simple/src/main/resources/archetype-resources/src/main/scala/__name__Service.scala
package ${package}

import com.yammer.dropwizard.ScalaService
import com.yammer.dropwizard.config.Environment

object ${name}Service 
  extends ScalaService[${name}Configuration]("${name}") {

  def initialize(conf: ${name}Configuration, env: Environment) {
    // TODO: implement service
  }

}

